docs/library/machine.Timer.rst

```diff
@@ -27,10 +27,11 @@ instead of this class.
 Constructors
 ------------
 
-.. class:: Timer(id, ...)
+.. class:: Timer(id)
```

Comment:
This conflicts with https://github.com/micropython/micropython/pull/6349

It looks like additional keyword arguments are allowed here.

Reply:
This is what I get on ESP32:

```
>>> import machine
>>> machine.Timer(0, mode=machine.Timer.PERIODIC)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: function doesn't take keyword arguments
>>> machine.Timer(0, machine.Timer.PERIODIC)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: function takes 1 positional arguments but 2 were given
>>> machine.Timer(0)
Timer(3ffe5590; alarm_en=0, auto_reload=1, counter_en=0)
```